docs/FAQ/gui-faq-sv.md

```diff
@@ -3,146 +3,199 @@ Zonemaster
+En beskrivning av felnivåerna som används, som *notice* ("varsel"), *warning*
+("varning") och *error* ("fel") finns i dokumentet "[Severity Level Definitions]"
+(*Defintion av felnivåer*).
+
+Ibland finns det olika tolkningar av standarden eller synpunkter på vad som är
+den rekommenderade inställningen. Zonemastergruppen välkomnar alltid synpunkter
+och kommentarer på detta. Om du tror du hittat något som du tycker att vi har
+felbedömt, tveka inte att kontakta oss på
 [zonemaster-users@lists.iis.se](mailto:zonemaster-users@lists.iis.se) (modererad
 e-postlista) med en länk till ditt test och en förklaring av varför du anser att
 resultatet inte är korrekt.
 
 <a name="q6"></a>
 #### 6. Kan Zonemaster hantera IPv6?
 
-Ja. Alla tester som körs över IPv4 kommer även köras över IPv6 om Zonemaster är konfigurerad att göra det.
+Ja. Zonemaster kommer att skicka DNS frågor till namnservrarna över både IPv4
+och IPv6 om inte Zonemaster har konfigurerats på annat sätt. Under knappen
+"alternativ" så går det att ställa in detta.
 
 <a name="q7"></a>
-#### 7. Kan Zonemaster hantera DNSSEC?
+#### 7. Kan Zonemaster kontrollera DNSSEC?
 
 Ja. Om en domän som testas av Zonemaster har DNSSEC konfigurerat så kommer det testas automatiskt.
 
 <a name="q8"></a>
-#### 8. Vad skiljer Zonemaster från annan mjukvara som testar domäner?
-För det första så sparar Zonemaster all testhistoria. Det innebär att du kan gå tillbaka och titta på ett test du gjorde för en vecka sedan och jämföra det med ett test du nyss gjorde.
+#### 8. Vad skiljer Zonemaster från andra program som testar domäner?
+För det första så sparar Zonemaster all testhistorik utifrån den testade
+domänen, vilket innebär att man kan gå tillbaka till ett tidigare test och
+jämföra med ett test som just har körts.
 
-Alla test som Zonemaster kör är definierade i testfallsspecifikationer som
-finns länkade från "[Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md)".
+För det andra så är alla test som Zonemaster kör specificerade i
+"[Test Case Specifications][Defined Test Cases]" (*testfallsspecifikationer*).
 
-Zonemaster kan också testa icke-publicerade/odelegerade domäner (mer om detta i [fråga 12](#q12)).
+För det tredje så kan Zonemaster också testa icke-publicerade eller odelegerade
+domäner (se [fråga 12][Question 12]).
 
-Zonemaster är dessutom öppen källkod och är modulärt uppbyggd. Du kan med andra ord återanvända
-hela eller delar av koden i dina egna system om du vill.
+För det fjärde så kan Zonemaster användas för att testa DS-posterna innan de
+läggs in i moderzonen, vilket är nödvändigt för att aktivera DNSSEC för en
+signerad zon (se [fråga 13][Question 13]).
+
+Och till sist, Zonemaster är öppen källkod och är modulärt uppbyggt, vilket
+betyder att man kan integrera delar av Zonemaster i sitt eget system om man
+vill. Till exempel så är det ovanligt att man vill ha ett komplett program bara
```

Comment:
Förstår inte. Är ZM lösningen på ett ovanligt problem?

Reply:
This is a translation of a strange sentence in the English version that I do not fully understand:

> For example, it is quite rare that you would want a complete program just to check for
> redelegations.

I remove it from the Swedish text.